pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -94,12 +93,29 @@ func (o *BuildChainOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, a
 	}
 	o.c, o.t = oc, oc
 
-	// Parse user input
-	o.name, o.tag, err = buildChainInput(args[0])
-	if err != nil {
-		return cmdutil.UsageError(cmd, err.Error())
+	parts := strings.Split(args[0], "/")
```

Comment:
This seems like it changes the behavior of the lookup "namespace/name:tag" for ists.  So this breaks existing usage?